PORTAL/tests/helpers.py

```diff
@@ -0,0 +1,12 @@
+from pathlib import Path
+
+
+DATA_ROOT = Path(__file__).parent / "data"
+
+NEW_FILENAME = DATA_ROOT / "cpython-3.11.0b1-8d32a5c8c4-fc_linux-b2cf916db80e.json"
+
+CONFIG_FILENAME = DATA_ROOT / "jobs.json"
```

Comment:
Note that "jobs.json" already has meaning in this project, so consider something like "config.json" instead.

Reply:
I'm not opposed, but I think it's clearer this way, since `jobs.json` is the default name of the config file, and this is a test-specific replacement for it.

Reply:
Oops, I was thinking of something else (queue.json, I think).  I'm on board with matching whatever the file is named normally.